Workbooks/Azure Security Center/Containers Security/Containers Security.workbook

```diff
@@ -2231,748 +6349,747 @@
+              "style": "info"
             },
-            {
-              "columnId": "subAssessmentName",
-              "label": "Security check"
+            "conditionalVisibility": {
+              "parameterName": "acrItemSelected",
+              "comparison": "isEqualTo",
+              "value": "true"
             },
-            {
-              "columnId": "severity",
-              "label": "Severity"
-            }
-          ]
-        }
+            "name": "Acr_Vulnerability_card "
+          }
+        ]
       },
-      "conditionalVisibilities": [
-        {
-          "parameterName": "tab",
-          "comparison": "isEqualTo",
-          "value": "k8s"
-        },
-        {
-          "parameterName": "k8sSelectedVulnerability",
-          "comparison": "isEqualTo",
-          "value": "true"
-        }
-      ],
-      "name": "K8s_Containers_Affected_By_Vulnerability"
-    },
-    {
-      "type": 1,
-      "content": {
-        "json": "### Description \r\n{k8sSubAssessmentDescription}\r\n### Remediation\r\n{k8sSubAssessmentRemediation}\r\n### General Information\r\n<table style=\"width:100%\">\r\n  <tr>\r\n    <th>ID</th>\r\n    <th>Severity</th>\r\n    <th>Patchable</th>\r\n\t<th>Published</th>\r\n  </tr>\r\n    <tr>\r\n    <td>{k8sId}</td>\r\n    <td>{k8sSeverity}</td>\r\n    <td>{k8sPatchable}</td>\r\n\t<td>{k8sPublished}</td>\r\n  </tr>\r\n  </table>\r\n### Impact\r\n{k8sSubAssessmentImpact}\r\n\r\n",
-        "style": "info"
+      "conditionalVisibility": {
+        "parameterName": "tab",
+        "comparison": "isEqualTo",
+        "value": "acrmdvm"
       },
-      "conditionalVisibilities": [
-        {
-          "parameterName": "tab",
-          "comparison": "isEqualTo",
-          "value": "k8s"
-        },
-        {
-          "parameterName": "k8sSelectedContainer",
-          "comparison": "isEqualTo",
-          "value": "true"
-        }
-      ],
-      "name": "k8s_Vulnerability_Card"
+      "name": "acr-mdvm"
     }
   ],
+  "fallbackResourceIds": [
+    "Azure Security Center"
+  ],
+  "fromTemplateId": "community-Workbooks/Azure Security Center/Containers Security",
```

Comment:
@future-at-work The PR checks fail because of this line. You should remove it.

Reply:
All of 7090 to 7093 or just 7093?

Reply:
@liorarviv I figured out which line and removed it